showcase/src/app/demo-ui/pages/generic-search/components/demo-generic-search-form.component.html

```diff
@@ -0,0 +1,48 @@
+<div fxLayout="row wrap">
```

Comment:
So I guess all CSS about "stark-layout" should be removed as it is not used right?

Reply:
Currently, it's not used. So I remove it and we'll add it back if needed.